### PR TITLE
Fix incorrect precedence of pow operator in GDScript

### PR DESCRIPTION
**Pull request by Chaosus**
Fix incorrect precedence, eg:

```
func _ready():
	print(4 * 2 ** 5) # prints 32768, should be 128
```

(in the expressions, it's already working correctly)

### CODE REVIEW

File: modules/gdscript/gdscript_parser.h [hunk 1]
```diff
@@ -1265,6 +1265,7 @@ class GDScriptParser {
 		PREC_FACTOR,
 		PREC_SIGN,
 		PREC_BIT_NOT,
+		PREC_POWER,
```
Comment (akien-mga) [inline modules/gdscript/gdscript_parser.h:1268]: Shouldn't it be further up? This sounds like it has lower precedence as e.g. `PREC_ADDITION_SUBTRACTION`.

Comment (akien-mga) [reply modules/gdscript/gdscript_parser.h:1268]: Ah nevermind I read it wrong. "Higher value" obviously means defined _later_ in the enum.

**Review by akien-mga — APPROVED** [review]

File: modules/gdscript/gdscript_parser.h (hunk 1 above)
Comment (Chaosus) [reply modules/gdscript/gdscript_parser.h:1268]: No, the higher value here means high priority

Comment (Chaosus) [reply modules/gdscript/gdscript_parser.h:1268]: Indeed, :)

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**